charts/portainer/templates/deployment.yaml

```diff
@@ -62,26 +62,12 @@ spec:
             - name: http
               containerPort: 9000
               protocol: TCP
-            {{- if not .Values.enterpriseEdition.enabled }}
             - name: https
               containerPort: 9443
-              protocol: TCP
-            {{- end }}                
+              protocol: TCP                
```

Comment:
IIRC @deviantony added a PR recently to make 9443 specific to CE.. is this a regression, or an intended removal?

Reply:
intended, EE 2.10 adds the https 9443
It was CE only before.

Reply:
Seems sensible :)